ios/bundleReactNative.sh

```diff
@@ -6,13 +6,17 @@ export NODE_OPTIONS=--max_old_space_size=12000
 export BUNDLE_COMMAND="bundle"
 export ENTRY_FILE="index.ts"
 
+# Disable the below if you are not using Sentry
+export SENTRY_ENABLED=true
```

Comment:
This should be an env var, not to be set here

Reply:
Removed